nixos/modules/security/apparmor/includes.nix

```diff
@@ -62,7 +62,7 @@ config.security.apparmor.includes = {
     include "${pkgs.apparmor-profiles}/etc/apparmor.d/abstractions/base"
     r ${pkgs.stdenv.cc.libc}/share/locale/**,
     r ${pkgs.stdenv.cc.libc}/share/locale.alias,
-    ${lib.optionalString (pkgs.glibcLocales != null) "r ${pkgs.glibcLocales}/lib/locale/locale-archive,"}
+    r ${config.i18n.glibcLocales}/lib/locale/locale-archive,
```

Comment:
Slightly unrelated, but is there a reason we're using `r $path,` instead of `$path r`?

The second way is what I've seen everywhere else and iirc also what the profiles in `apparmor-profiles` contain.

Reply:
No idea why, I just followed the style that was already present.

But I think following conventions leads to less confusion. And we should not needlessly deviate.

When I find the time, I will swap the order.

Reply:
I think we can do that in a follow-up PR, if you agree.

Then I'd just merge this as-is and someone (maybe you, maybe me, maybe someone else) can switch all existing profiles in nixpkgs to the more conventional syntax, if there is not objection (e.g. by @ju1m).

Reply:
Sounds good to me :+1:

Reply:
@ajs124, @tnias, I used that syntax because it was easier for me to read (the permissions part being easier to find/review), but I agree with you that it may cause confusion for others and should be changed to the conventional syntax, or the [third syntax](https://gitlab.com/apparmor/apparmor/-/wikis/QuickProfileLanguage#file-rules), which is less terse but mixes the best of the other two:
```
    /path/to/file3       # file rule split over multiple lines
         rw,
```